README.md

```diff
@@ -20,6 +20,9 @@ The LogDNA AWS CloudWatch integration relies on [AWS Lambda](https://aws.amazon.
     * `LOGDNA_HOSTNAME`: Alternative Host Name *(Optional)*  
     * `LOGDNA_TAGS`: Comma-separated Tags *(Optional)*  
     * `LOGDNA_URL`: Custom Ingestion URL *(Optional)*
+    * `RAW_EVENTLOG`: Setting `line` to Raw `event.message` *(Optional, Default: false)*:
```

Comment:
I still don't think this variable name matches up with the expected type. It's still a noun.

Reply:
I see now the noun ones may get the actual event message put as a value by the customer - changed it!